sources/packages/web/src/views/aest/student/FileUploads.vue

```diff
@@ -1,156 +1,38 @@
-        :formData="initialData"
-        formName="uploadStudentDocumentsAEST"
-      >
-        <template #actions="{ cancel, submit }">
-          <v-row class="m-0 p-0">
-            <v-btn color="primary" variant="outlined" @click="cancel"
-              >Cancel</v-btn
-            >
-            <check-permission-role :role="Role.StudentUploadFile">
-              <template #="{ notAllowed }">
-                <v-btn
-                  class="float-right"
-                  @click="submit"
-                  color="primary"
-                  variant="elevated"
-                  :disabled="notAllowed"
-                  >Upload now</v-btn
-                >
-              </template>
-            </check-permission-role>
-          </v-row>
-        </template>
-      </formio-modal-dialog>
-    </body-header-container>
+    <student-file-uploads
+      :studentId="studentId"
+      :canUploadFiles="true"
+      :canDownloadFiles="true"
+      @uploadFile="uploadFile"
+    >
+    </student-file-uploads>
   </tab-container>
 </template>
 
 <script lang="ts">
-import { onMounted, ref, defineComponent } from "vue";
-import {
-  DEFAULT_PAGE_LIMIT,
-  FormIOForm,
-  PAGINATION_LIST,
-  LayoutTemplates,
-  Role,
-} from "@/types";
+import { defineComponent } from "vue";
+import { useFormioUtils, useSnackBar } from "@/composables";
+import StudentFileUploads from "@/components/common/students/StudentFileUploads.vue";
 import { StudentService } from "@/services/StudentService";
-import {
-  useFormatters,
-  useFileUtils,
-  ModalDialog,
-  useFormioUtils,
-  useSnackBar,
-} from "@/composables";
-import FormioModalDialog from "@/components/generic/FormioModalDialog.vue";
-import {
-  AESTFileUploadToStudentAPIInDTO,
-  StudentUploadFileAPIOutDTO,
-} from "@/services/http/dto/Student.dto";
-import CheckPermissionRole from "@/components/generic/CheckPermissionRole.vue";
+import { AESTFileUploadToStudentAPIInDTO } from "@/services/http/dto/Student.dto";
 
 export default defineComponent({
-  components: {
-    FormioModalDialog,
-    CheckPermissionRole,
-  },
+  components: { StudentFileUploads },
   props: {
     studentId: {
       type: Number,
       required: true,
     },
   },
-  setup(props: any) {
-    const studentFileUploads = ref([] as StudentUploadFileAPIOutDTO[]);
-    const fileUploadModal = ref({} as ModalDialog<FormIOForm | boolean>);
-    const { dateOnlyLongString } = useFormatters();
+  setup(props) {
     const snackBar = useSnackBar();
-    const fileUtils = useFileUtils();
     const formioUtils = useFormioUtils();
-    const initialData = ref({ studentId: props.studentId });
-    const loadStudentFileUploads = async () => {
-      studentFileUploads.value =
-        await StudentService.shared.getAESTStudentFiles(props.studentId);
-    };
-
-    onMounted(async () => {
-      await loadStudentFileUploads();
-    });
 
-    const uploadFile = async () => {
+    const uploadFile = async (
+      fileUploadModal,
```

Comment:
Please check if you have the below waning, we should get rid of them.
![image](https://user-images.githubusercontent.com/61259237/234081710-076654d2-122b-4044-bf54-aca08adc82dc.png)